### PR TITLE
Adding User Group and some Endpoints

### PR DESCRIPTION
**Pull request by binghamtj**
@ejcarr @DudeSolutions/software-engineering @brianhyder

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -156,8 +191,8 @@ Any response that does not provide a ```201 Created``` response should be treate
 
 + Response 204
 
-    
-### 
+
+###
```
Comment (elizabethmidgett) [inline apiary.apib]: I see this was previously in the file, but is it needed?

File: apiary.apib [hunk 2]
```diff
@@ -71,6 +71,41 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+
++ Response 201 (application/json)
+  + Attributes (User)
+
+### DELETE
+
++ Response 201
+  + Attributes (User)
+
+###
```
Comment (brianhyder) [inline apiary.apib]: What is this here for?

Comment (binghatz) [inline apiary.apib]: I thought I needed to document the endpoints for the SDK to use.

File: apiary.apib (hunk 1 above)
Comment (binghatz) [inline apiary.apib]: I don't know, I saw it done previously and copied.

File: apiary.apib (hunk 2 above)
Comment (binghamtj) [inline apiary.apib]: Removed the ###

File: apiary.apib (hunk 1 above)
Comment (binghamtj) [inline apiary.apib]: Removed the ###

**Pull request merged**